### PR TITLE
added get_channel_id method

### PR DESCRIPTION
**Pull request by sumzer**
We are using this repo for our project and we need method getChannelId wich is provided by UrbanAirship SDk library.
http://docs.urbanairship.com/platform/android.html#retrieving-your-channel-id
can you check it and approve it.

Thanks in advance
Best regards 
Sumi

### CODE REVIEW

File: android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java [hunk 1]
```diff
@@ -55,6 +55,12 @@ public void removeTag(String tag) {
     }
 
     @ReactMethod
+    public void getChannelId(Callback callback) {
+	     String channelId = UAirship.shared().getPushManager().getChannelId();
```
Comment (rlepinski) [inline android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java]: spacing is off here

**Comment by rlepinski** [comment]
Could you update the README with the added method?

Other than the code spacing issue, everything looks good to me.

**Comment by sumzer** [comment]
Hi,
I've pushed changes, please check it.
Best regards
Sumi

**Comment by rlepinski** [comment]
This looks great to me. I do not have the ability to merge commits. Paging @evandroeisinger

**Comment by evandroeisinger** [comment]
@rlepinski you didn't have? We already gave you write permissions; Could you check, please?

**Comment by rlepinski** [comment]
@evandroeisinger I had an invite in my inbox from a few weeks ago. Thanks :)

**Comment by evandroeisinger** [comment]
@rlepinski just to let you know that I already added you to the npm project. Now you are able to deploy new versions.

**Pull request merged**